CMakeLists.txt

```diff
@@ -1,39 +1,20 @@
 cmake_minimum_required(VERSION 3.0)
```

Comment:
should have updated the version number, to get a better message than "Missing function or something like that

Reply:
Updated